app/views/layouts/base.html.erb

```diff
@@ -15,7 +15,19 @@
   <meta content="noindex,nofollow" name="robots" />
   <% end %>
 
-  <title><%= yield(:title).presence || (raise 'Missing title') %> | <%= APP_NAME %></title>
+  <%
+    title = begin
+      yield(:title).presence || (raise 'Missing title')
+    rescue => err
```

Comment:
And if we did move the logic elsewhere, maybe we could create a custom Error subclass to filter the rescue?

Reply:
We could create an error class if we want to anyways? I'm not worried that this fairly tightly-scoped `begin...end` is going to eat extra errors